localstack/config.py

```diff
@@ -53,6 +53,9 @@
 
 # folder for temporary files and data
 TMP_FOLDER = os.path.join(tempfile.gettempdir(), 'localstack')
+# fix for Mac OS, to be able to mount /var/folders in Docker
```

Comment:
Lines 56-58 should be _below_ the folder creation block in lines 195-203. Can we move that block up here, and hence also use the definition of `DATA_DIR` further above?

The reason is that the code can otherwise fail under certain circumstances, particularly on the FIRST startup of LocalStack (i.e., when the folder `os.path.join(tempfile.gettempdir(), 'localstack')`) does not yet exist yet. Hence lines 56-58 should be _below_ 195-203

Reply:
Moved the folder creation up. It works but I am not sure what other effects this change may have.